MobiusCore/Source/AsyncStartStopStateMachine.swift

```diff
@@ -39,6 +39,10 @@ final class AsyncStartStopStateMachine<StoppedState, RunningState> {
         case transitioningToStopped
     }
 
+    enum Error: Swift.Error {
```

Comment:
How about using a struct instead? Eg

```swift
struct WrongStateError: Error {}

// throw WrongStateError()
```

Reply:
I feel a singleton enum is clearer/less unexpected than a singleton struct for this case, but I’ll change the string wrapper one.